packages/components/src/Toast/Toast.tsx

```diff
@@ -9,7 +9,7 @@ import { Typography } from "../Typography";
 interface BaseToastProps {
   readonly variation?: "info" | "success" | "error";
   readonly message: string;
-  readonly id?: number;
+  readonly id?: string;
```

Comment:
I think we need to support backwards compatibility by doing `string | number`?

Reply:
I thought about that but I don't see it being used anywhere in JO (I opened a PR to see if any errors appear in CI, but I also search for all references of `id` and nothing). If you mean in terms of 3rd parties using this and us breaking it, that's fair, but is it worth it? Cause I doubt this will affect anyone either.

Reply:
That's probably right, but I'll let Atlantis folks weigh in.

Reply:
Yeah, thanks for making this backward compatible. We use semantic versioning and this would be a breaking change.